snippets/csharp/programming-guide/discards/discard-pattern2.cs

```diff
@@ -21,7 +21,7 @@ private static void ProvidesFormatInfo(object obj)
          Console.Write("A null object reference: ");
          Console.WriteLine("Its use could result in a NullReferenceException");
       }
-      else if (obj is var _)
+      else
```

Comment:
Although this change is acceptable, it requires a change to example description. The example is used in [Discards - C# Guide](https://docs.microsoft.com/en-us/dotnet/csharp/discards), where the sentence "It also used the discard pattern to handle non-null objects of any other type." needs to be removed. Could you open a PR to do that?